bin/i18n/resources/dashboard/courses/sync_out.rb

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env ruby
+
+require_relative '../../../i18n_script_utils'
+require_relative '../../../redact_restore_utils'
+require_relative '../../../utils/sync_out_base'
+require_relative '../../../utils/malformed_i18n_reporter'
+require_relative '../courses'
+
+module I18n
+  module Resources
+    module Dashboard
+      module Courses
+        class SyncOut < I18n::Utils::SyncOutBase
+          def process(language)
+            crowdin_file_path = crowdin_file_path_of(language)
+            return unless File.file?(crowdin_file_path)
+
+            unless I18nScriptUtils.source_lang?(language)
+              restore_localization(language)
+              fix_localization_urls(language)
+              report_malformed_i18n(language)
+              distribute_localization(language)
+            end
+
+            i18n_file_path = I18nScriptUtils.locale_dir(language[:locale_s], DIR_NAME, FILE_NAME)
+            I18nScriptUtils.move_file(crowdin_file_path, i18n_file_path)
+          end
+
+          private
+
+          def crowdin_file_path_of(language)
+            I18nScriptUtils.locale_dir(language[:crowdin_name_s], DIR_NAME, FILE_NAME)
+          end
+
+          def restore_localization(language)
+            return unless File.exist?(I18N_BACKUP_FILE_PATH)
+
+            crowdin_file_path = crowdin_file_path_of(language)
+
+            RedactRestoreUtils.restore(
+              I18N_BACKUP_FILE_PATH,
+              crowdin_file_path,
+              crowdin_file_path,
+              REDACT_PLUGINS,
+              REDACT_FORMAT
+            )
+          end
+
+          # We provide URLs to the translators for Resources only; because
+          # the sync has a side effect of applying Markdown formatting to
+          # everything it encounters, we want to make sure to un-Markdownify
+          # these URLs
+          def fix_localization_urls(language)
```

Comment:
This method was `def self.postprocess_course_resources(locale, courses_source)` before. 
The new name is more descriptive of what the function is doing ✨